hl-todo.el

```diff
@@ -232,6 +232,20 @@ matcher."
   (interactive)
   (occur hl-todo--regexp))
 
+;;;###autoload
+(defun hl-todo-insert-keyword ()
+  "A helper command to insert a TODO or similar keywords.
+All defined keywords are in `hl-todo-keyword-faces' alist."
+  (interactive)
+  (let ((keyword (completing-read "hl-todo keyword: "
```

Comment:
This should happen inside `interactive`.

Reply:
Hmm, right, let me update the code.